packages/desktop/App.svelte

```diff
@@ -104,9 +104,10 @@
             addError(err)
         })
 
+        cleanupInProgressProfiles()
+
```

Comment:
Was this moved on purpose and is it no longer needer to `await` for a promise?

Reply:
Yes I believe it was typed as returning a `Promise` but nothing actually required that.